src/front/spv/mod.rs

```diff
@@ -2222,6 +2228,29 @@ impl<I: Iterator<Item = u32>> Parser<I> {
             self.index_constants.push(handle);
         }
 
+        // register maths constants
+        self.maths_constants.clear();
+        {
+            self.maths_constants
+                .push(module.constants.append(crate::Constant {
+                    name: None,
+                    specialization: None,
+                    inner: crate::ConstantInner::Scalar {
+                        width: 4,
+                        value: crate::ScalarValue::Float(std::f64::consts::PI / 180.0),
+                    },
+                }));
+            self.maths_constants
```

Comment:
given that this is a very narrow use case, can we get away with 1 constant instead of 2?

Reply:
It's not a huge deal, but IMO the resulting shaders are going to be slightly nicer if we use 2 constants and multiply by both, rather than having 1 and choosing to multiply or divide.